bokeh/models/widgets/tables.py

```diff
@@ -58,6 +58,13 @@
     'TextEditor',
     'TimeEditor',
     'TableWidget',
+    'AvgAggregator',
+    'MinAggregator',
+    'MaxAggregator',
+    'SumAggregator',
+    'TableWidget',
```

Comment:
@jburgy there is already `TableWidget` in the list above, this is what is causing the docs failure. Also the `__all__` list should generally be sorted alphabetically.